SECURITY.md

```diff
@@ -9,10 +9,10 @@
 
 ## Reporting a Vulnerability
 
-Vulnerabilities should ideally be reported by directly messaging one of the maintainers on Discord. Maintainers can be found in the [#coderbus](https://discord.gg/Vh8TJp9) guild in the [#hosting-questions](https://discord.com/channels/484170914754330625/653425022966169620) channel. Please be sure to provide reproduction steps.
+Vulnerabilities should ideally be reported by directly messaging one of the TGS maintainers on Discord. Maintainers can be found in the [#coderbus](https://discord.gg/Vh8TJp9) guild in the [#hosting-questions](https://discord.com/channels/484170914754330625/653425022966169620) channel. Please be sure to provide reproduction steps.
 
 Here is a list of their discord IDs.
 
-@Cyberboss - Dominion#0444
+@Cyberboss - Dominion#0444 (<@133295178197893120>)
```

Comment:
```suggestion
@Cyberboss - Dominion#0444
```

Only way to find a user by ID is with a bot so.... vOv

Reply:
You can type in <@userid> and it will replace the userid with a proper mention

Reply:
https://user-images.githubusercontent.com/25136265/184975992-268c57d1-6f41-4e8c-9327-38a77af488a3.mp4